cves/kernel/CVE-2013-2017.yml

```diff
@@ -1,288 +1,196 @@
-      - Bug labels
-      - Directory names
-      - How developers refer to an area of the system in comments,
-        commit messages, etc.
-
-    Look at the path of the source code files code that were fixed to get
-    directory names. Look at comments in the code. Look at the bug reports how
-    the bug report was tagged.
-
-    Example linux kernel subsystems are:
-      * drivers
-      * crypto
-      * fs
-      * net
-      * lib
-
-    Name should be:
-      * all lowercase English letters
-      * NOT a specific file
-      * can have digits, and _-@/
-
-    Can be multiple subsystems involved, in which case you can make it an array
-    e.g.
-        name: ["subsystemA", "subsystemB"] # ok
-        name: subsystemA # also ok
-  name:
-  note:
-interesting_commits:
-  question: |
-    Are there any interesting commits between your VCC(s) and fix(es)?
-
-    Use this to specify any commits you think are notable in some way, and
-    explain why in the note.
-
-    Example interesting commits:
-      * Mentioned as a problematic commit in the past
-        e.g. "This fixes regression in commit xys"
-      * A significant rewrite in the git history
-      * Other commits that fixed a similar issue as this vulnerability
-      * Anything else you find interesting.
-  commits:
-  - commit:
-    note:
-  - commit:
-    note:
-i18n:
-  question: |
-    Was the feature impacted by this vulnerability about internationalization
-    (i18n)?
-
-    An internationalization feature is one that enables people from all
-    over the world to use the system. This includes translations, locales,
-    typography, unicode, or various other features.
-
-    Answer should be true or false
-    Write a note about how you came to the conclusions you did, regardless of
-    what your answer was.
-  answer:
-  note:
-sandbox:
-  question: |
-    Did this vulnerability violate a sandboxing feature that the system
-    provides?
-
-    A sandboxing feature is one that allows files, users, or other features
-    limited access. Vulnerabilities that violate sandboxes are usually based on
-    access control, checking privileges incorrectly, path traversal, and the
-    like.
-
-    Answer should be true or false
-    Write a note about how you came to the conclusions you did, regardless of
-    what your answer was.
-  answer:
-  note:
-ipc:
-  question: |
-    Did the feature that this vulnerability affected use inter-process
-    communication? IPC includes OS signals, pipes, stdin/stdout, message
-    passing, and clipboard. Writing to files that another program in this
-    software system reads is another form of IPC.
-
-    Answer must be true or false.
-    Write a note about how you came to the conclusions you did, regardless of
-    what your answer was.
-  answer:
-  note:
+  answer: |
+    This vulnerability was discovered by a developer, Martin Ferrari, when he noticed that the vulnerability caused him 
```

Comment:
Be more descriptive about how this was found. Looking at the description, I saw that the developer had a few steps in place to replicate the vulnerability. Be specific about that.